docs/user/dashboard/tutorial-create-a-dashboard-of-lens-panels.asciidoc

```diff
@@ -339,23 +272,52 @@ image::images/lens_end_to_end_6_1.png[Custom ranges configuration]
 
 .. From the *Value format* dropdown, select *Bytes (1024)*, then click *Close*.
 
-. From the *Chart Type* dropdown, select *Pie*.
+. Click *Bar vertical stacked*, then select *Pie*.
 +
 [role="screenshot"]
 image::images/lens_end_to_end_6_2.png[Files size distribution]
```

Comment:
Not super important, but I think it may be nice to have a small hint here about other types of charts.

Something like:
> .. Notice how the suggestion system is also recommending other types of visualization for the same dataset, as the treemap one. Treemaps are a convenient alternative to pie charts.

Reply:
@dej611 I would like to add more context in other places, but for this use case of showing two slices I think a pie chart is the only chart type I would use. The treemap here is pretty hard to read, in my opinion:

<img width="287" alt="Screen Shot 2021-04-21 at 3 24 57 PM" src="https://user-images.githubusercontent.com/666475/115609627-c69eac80-a2b5-11eb-9714-14c32a8a28c7.png">

Reply:
I do not find it too hard to read, but I guess it's subjective.
Anyway no strong opinion. The goal was more to hint the user to experiment with the Lens suggestion system as well rather than the specific case.